### DISCUSSION TITLE
Plotting interaction graphs from docking results

### DISCUSSION THREAD

**Discussion by Sumbul1631** [chemosim-lab/ProLIF#141] [General]
can i plot a graph of my interaction fingerprints from docking results.. something like this:

![Protein-ligand-interaction-fingerprint-PLIF-of-the-docking-poses-in-3G5U-Pgp](https://github.com/chemosim-lab/ProLIF/assets/139619428/249aaf85-c25f-4947-8805-32b270211b4d)

**Comment by cbouy** [comment]
Once you convert the ProLIF fingerprint object to a pandas DataFrame with `df = fp.to_dataframe()` you can transform the data and make this sort of plots for sure, but I don't have time to offer this kind of personalized support unfortunately.
I'll convert this to a discussion since it's not really an issue with the library itself.
